packages/sonarwhal/src/lib/cli/rules/templates/new-rule/tsconfig.json.hbs

```diff
@@ -0,0 +1,40 @@
+{{#if official}}{
+    "compilerOptions": {
+        "outDir": "dist"
+    },
+    "exclude": [
+        "dist",
+        "node_modules"
+    ],
+    "extends": "../../tsconfig.json",
+    "include": [
+        "src/**/*.ts",
+        "tests/**/*.ts"
+    ]
+}{{else}}{
+    "compilerOptions": {
+        "alwaysStrict": true,
+        "declaration": true,
+        "lib": [
+            "dom",
+            "dom.iterable",
+            "es2017",
+            "esnext",
+            "esnext.asynciterable"
+        ],
+        "module": "commonjs",
+        "newLine": "lf",
+        "outDir": "dist",
+        "removeComments": true,
+        "sourceMap": true,
```

Comment:
Shoundn't this be removed?

Reply:
Should be replaces with `inlineSourceMap`